scripts/synthdnn/synthdnn.py

```diff
@@ -113,6 +113,8 @@ def collect_main(args):
 
     if args.collect == "corr":
         benchdnn_args = f"--engine={args.engine} --matmul --mode-modifier=P {get_optional_args(args)}"
+    elif args.collect == "run":
+        benchdnn_args = f"--engine={args.engine} --matmul --mode=R --mode-modifier=P {get_optional_args(args)}"
     elif args.collect == "perf":
         benchdnn_args = f"--engine={args.engine} --matmul --mode=F --cold-cache=all --perf-template=sample,{args.name},%prb%,%0Gflops%,%0Gbw% --memory-kind=usm_device --attr-scratchpad=user {get_optional_args(args)}"
```

Comment:
Collect is mostly a wrapper around benchdnn. It's core purpose is to set a standardized output data format (i.e. perf-template) for processing by later scripts. Because of this, it likely makes more sense just forward (non-format related) command line arguments to benchdnn, rather than maintain a second interface.